### PR TITLE
Added all options

### PR DESCRIPTION
**Pull request by qwibbler**
Make CLI with options

- [ ]     List all books.
- [ ]     List all people.
- [ ]     Create a person (teacher or student, not a plain Person).
- [ ]     Create a book.
- [ ]     Create a rental.
- [ ]     List all rentals for a given person id.
- [ ]     Exit

### CODE REVIEW

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# STATUS: INVALID 🟡 
Hi @qwibbler, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 😢 

You have submitted a project, that linters are failing, please fix this before you send it for another review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

programming-punching-keyboard-coding-anger-13716030304


# CHEERS AND HAPPY CODING!! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

Invalid Code Review Request **does not count** into the code reviews limit.

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  end
+
+  def create_student(age, name)
+    print 'Has Parent Permission? [Y/N]: '
+    pp = gets.chomp
+    return wrong_option unless %w[y n].include? pp.downcase
+
+    @people << Student.new(age, name, parent_permission: pp.downcase == 'y')
+    'Student created successfully'
+  end
```
Comment (er-contreras) [inline main.rb]: - [x] When creating a student the output is not correct as you can see in the next image:

Screen Shot 2022-03-03 at 19 53 54

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @qwibbler, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by er-contreras — APPROVED** [review]
Hi @qwibbler, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**